app/ota_client.py

```diff
@@ -18,11 +18,14 @@
 from requests.exceptions import RequestException
 from retrying import retry
 
-from ota_status import OtaStatusControl, OtaStatus
+from extlinux_control import CBootControl, CBootControlMixin
+from ota_client_interface import OtaClientInterface
+from grub_ota_partition import GrubControlMixin, OtaPartitionFile
```

Comment:
`from grub_ota_partition import GrubControlMixin, OtaPartitionFile` can be remove to https://github.com/tier4/ota-client/pull/83/files#diff-4e43d11537bf0064fc07412772bff9e85eae2bc9334ee70950cd7217088600f0R746
not to create unintended config by `config = create_config(_detect_platform())`

Reply:
Update in commit cdf6688.
Right now platform specific modules will only be imported when needed.